examples/finnegans_wake_demo/finnegans-wake-demo-testnet.py

```diff
@@ -69,7 +69,7 @@
 
 # Then, there was bob. Bob learns about the
 # rest of the network from the seednode.
-bob = Bob(domain=TESTNET)
+bob = Bob(domain=TESTNET, provider_uri=PROVIDER_URI)
```

Comment:
As a design goal, Bob can operate entirely offchain without an ethereum provider connection.  If this is a required parameter, we'll need to take a deeper look an make it optional!